crates/compiler/gen_wasm/src/low_level.rs

```diff
@@ -1960,7 +1960,61 @@ impl<'a> LowLevelCall<'a> {
                 backend.storage.load_symbols(code_builder, self.arguments);
             }
 
-            PtrWrite => todo!("{:?}", self.lowlevel),
+            PtrStore => {
+                // PtrStore : Ptr a, a -> {}
+                let ptr = self.arguments[0];
+                let value = self.arguments[1];
+
+                let (ptr_local_id, offset) = match backend.storage.get(&ptr) {
+                    StoredValue::Local { local_id, .. } => (*local_id, 0),
+                    _ => internal_error!("A pointer will always be an i32"),
+                };
```

Comment:
this one is weird. This `storage.get`  gets the correct local (the `hole` symbol, defined by the join point. But `ensure_value_has_local`  instead gives `LocalId(3)` , which in practice corresponds with the frame pointer. I don't know why, but this works. 

is the fact that the symbol comes from a join point relevant (because that is kind of brittle). again, should we consider options beside `StoredValue::Local`?